geometry_msgs/Point point
bool hold_final_pose
---

---
